OpenRA.Mods.Common/Traits/Buildings/RallyPoint.cs

```diff
@@ -40,6 +40,8 @@ public class RallyPoint : IIssueOrder, IResolveOrder, ISync, INotifyOwnerChanged
 		public RallyPointInfo Info;
 		public string PaletteName { get; private set; }
 
+		private const uint ExtraDataSetPrimaryBuilding = 1;
```

Comment:
Any specific reason why this constant exists? It is a private constant used in 2 places so I would think it would be just as good to just put the 1 straight into the code.